docs/resources/managed_user.md

```diff
@@ -6,6 +6,9 @@ subcategory: "User"
 Provides an Artifactory managed user resource. This can be used to create and maintain Artifactory users. For example, service account where password is known and managed externally.
 
 Unlike `artifactory_unmanaged_user` and `artifactory_user`, the `password` attribute is required and cannot be empty.
+Consider using a separate provider to generate and manage passwords. 
+
+~> The password is stored in the Terraform state file. Make sure you secure it. 
```

Comment:
Let's link to this https://developer.hashicorp.com/terraform/language/state/sensitive-data